ext/storage_sources/azure_store/test/test_azure_connection.cpp

```diff
@@ -72,6 +72,7 @@ TEST_CASE("Testing Azure Connection Class", "azure-connection")
     auto azure_client = Azure::Storage::Blobs::BlobContainerClient::CreateFromConnectionString(
       std::getenv("AZURE_STORAGE_CONNECTION_STRING"), "myblobcontainer1");
     bool exists = false;
+    size_t object_size = 0;
```

Comment:
Should we be adding some tests around object size? Or is that a separate ticket

Reply:
added check in the `object_exists` unit testing